From d3494e906048a8e5882be0f63b44a6093c01105f Mon Sep 17 00:00:00 2001
From: Sanjeev Gupta <ghane0@gmail.com>
Date: Tue, 29 Nov 2016 22:14:19 +0800
Subject: [PATCH 068/268] "Force update on NTPsec website"

Some docs were updated in mid-Nov, but not rebuilt
on the www host.  Touching them again to force a rebuild
---
 docs/includes/refclock.txt | 2 +-
 docs/rdebug.txt            | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/docs/includes/refclock.txt b/docs/includes/refclock.txt
index c212e49..d4cc0d2 100644
--- a/docs/includes/refclock.txt
+++ b/docs/includes/refclock.txt
@@ -1,6 +1,6 @@
 === Reference Clock Support ===
 * link:extern.html[External Clock Discipline and the Local Clock Driver]
-* link:driver_howto.html[How to Write a Reference Clock Driver]
+* link:driver_howto.html[How to Write a Reference Clock Driver] 
 * link:generic_howto.html[How to build new generic clocks]
 * link:refclock.html[Reference Clock Drivers]
 * link:sitemap.html[Site Map]
diff --git a/docs/rdebug.txt b/docs/rdebug.txt
index 998d15c..48c1e57 100644
--- a/docs/rdebug.txt
+++ b/docs/rdebug.txt
@@ -11,7 +11,7 @@ Call the girls and they'll sweep your bugs.
 
 == Related Links ==
 
-include::includes/refclock.txt[]
+include::includes/refclock.txt[] 
 include::includes/install.txt[]
 
 '''''
-- 
2.7.4

